packages/augur-test/src/tests/connector/seo-connector.test.ts

```diff
@@ -120,11 +120,12 @@ test('SEOConnector :: Should route correctly and handle events', async done => {
       expect(arg).toEqual(
         {
           eventName: SubscriptionEventName.NewBlock,
-          blocksBehindCurrent: 0,
+          blocksBehindCurrent: expect.any(Number),
           highestAvailableBlockNumber: expect.any(Number),
           lastSyncedBlockNumber: expect.any(Number),
-          percentBehindCurrent: '0.0000',
-        },
+          percentBehindCurrent: expect.any(String),
+          timestamp: expect.any(Number),
```

Comment:
Do we have enough control to use specific values here to check calculations?

Reply:
Sadly no because events come in during processing. The values keep changing and it's ephemeral (timestamp for example) so I can't nail them down.